torchgeo/losses/ssl.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Loss functions self-supervised learning."""
+
+import torch
+import torch.nn.functional as F
+from torch.nn.modules import Module
```

Comment:
```suggestion
from torch.nn import Module
```
I try to import things at the same level that they get imported in the docs